### PR TITLE
Adds OSS dm and removes fixed versions

### PR DESCRIPTION
**Pull request by joaofreitas-qubit**
Relates with: #qubIT-Omnis-4736

### CODE REVIEW

**Review by joaojtmarques-qubit — APPROVED** [review]
Approved after successful merge, as conflicts will need to be resolved

**Review by pabrantes — APPROVED** [review]

**Pull request merged**
